apps/aehttp/test/aehttp_contracts_SUITE.erl

```diff
@@ -1277,8 +1276,6 @@ args_to_list([]) -> [].
 
 %%arg_to_list(<<N:256>>) -> integer_to_list(N);
 arg_to_list(N) when is_integer(N) -> integer_to_list(N);
-arg_to_list(<<$",_/binary>>=B) -> 		%A string
```

Comment:
I see this is the main change.

I understand [a sample symptom](https://circleci.com/gh/aeternity/epoch/23153) was:
```
%%% aehttp_contracts_SUITE ==> contracts.environment_contract: FAILED
%%% aehttp_contracts_SUITE ==> {{badmatch,{error,<<"argument_syntax_error">>}},
 [{aehttp_contracts_SUITE,call_func,8,
                          [{file,"/home/builder/epoch/_build/test/lib/aehttp/test/aehttp_contracts_SUITE.erl"},
                           {line,949}]},
  {aehttp_contracts_SUITE,environment_contract,1,
                          [{file,"/home/builder/epoch/_build/test/lib/aehttp/test/aehttp_contracts_SUITE.erl"},
                           {line,668}]},
  {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1546}]},
  {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1062}]},
  {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,994}]}]}
```
i.e. on `args_to_binary([BPubkey])`.

So I infer root cause was byte `,` in randomly generated pubkey?

Reply:
And if a non-string binary happened to start with a 34 ("). String also need more work to handle if they include a ", this I will fix when I need it.